unresolved external symbols of winver.h

[These](https://github.com/oracle/graal/commit/29f31eaaf23b4d77d5f96aafb9840cae882edc04) changes to GraalVM introduce calls to `winver.h`, which causes these exception when linking:

 ```
[Thu Dec 10 13:34:31 CET 2020][INFO] [SUB] imagename.obj : error LNK2001: unresolved external symbol GetFileVersionInfoSizeW
[Thu Dec 10 13:34:31 CET 2020][INFO] [SUB] imagename.obj : error LNK2001: unresolved external symbol GetFileVersionInfoW
[Thu Dec 10 13:34:31 CET 2020][INFO] [SUB] imagename.obj : error LNK2001: unresolved external symbol VerQueryValueW
```

--- a/src/main/java/com/gluonhq/substrate/target/WindowsTargetConfiguration.java
+++ b/src/main/java/com/gluonhq/substrate/target/WindowsTargetConfiguration.java
@@ -41,7 +41,7 @@ import java.util.stream.Collectors;
 public class WindowsTargetConfiguration extends AbstractTargetConfiguration {
 
     private static final List<String> javaWindowsLibs = Arrays.asList(
-            "advapi32", "iphlpapi", "secur32", "userenv", "ws2_32");
+            "advapi32", "iphlpapi", "secur32", "userenv", "version", "ws2_32");
     private static final List<String> staticJavaLibs = Arrays.asList(
             "j2pkcs11", "java", "net", "nio", "prefs", "fdlibm", "sunec", "zip");
     private static final List<String> staticJvmLibs = Arrays.asList(

